### DISCUSSION TITLE
AOC Day 13 Discussion: Sorting it out

### DISCUSSION THREAD

**Discussion by rhdeck** [Virtual-Coffee/virtualcoffee.io#773] [Advent of Code 2022]
Discussion of solutions for [Advent of Code Day 13](https://adventofcode.com/2022/day/13) here.

Don't forget about the VC Advent of Code private leaderboard:

Go to [leaderboard/private](https://adventofcode.com/2022/leaderboard/private)
Use join code 363496-e6f42283

**Comment by rhdeck** [comment]
A task I could do cold in JS, and had to learn how to do it in python. functools ftw! [solve](https://github.com/rhdeck/aoc-2022/blob/main/src/13/day_13.ipynb)

**Reply by doleraj** [reply]
Yeah the choice of JS/TS today really made this just a matter of turning the crank.

**Reply by tkshill** [reply]
@rhdeck I feel like you're getting more and more concise with every problem!

**Comment by doleraj** [comment]
Huzzah for JSON.parse. https://github.com/doleraj/adventofcode/blob/main/aoc2022/src/day13/index.ts

**Reply by tkshill** [reply]
Vicariously learning about js through these

**Comment by tkshill** [comment]
Finally starting using my favourite parsing library. @doleraj made me realize I could just use a jsonparser, but I like the practice of working with fparsec.

The rest was just a bit of testing and pattern matching. 

This'll be the only time I ever enjoy spinning up a sorter.

[le solve](https://github.com/tkshill/AdventOfCode2022/blob/main/AOC2022/Day13.fs)
